Projects/Server/ExpansionInfo.cs

```diff
@@ -331,4 +263,23 @@ public static ExpansionInfo GetInfo(int ex)
 
         public override string ToString() => Name;
     }
+
+    public record ExpansionConfig
+    {
+        public string Name { get; init; }
+
+        [JsonConverter(typeof(ClientVersionConverter))]
```

Comment:
Client version can go into options. Only enum flags should be a custom attributed converter.